src/mixins/filing-template-mixin.ts

```diff
@@ -602,6 +621,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     // restore Dissolution data
     this.setBusinessAddress(draftFiling.dissolution.custodialOffice)
     this.setDissolutionType(draftFiling.dissolution.dissolutionType)
+    // setting completing party
+    this.setCompletingParty(this.completingParties(draftFiling.dissolution.parties))
```

Comment:
I was thinking this through and I think this should be _current_ CP, not the CP from last time. (Hence, the CP needs to be set every time this app starts.)

Thoughts?

Reply:
Also, where do you store the CP in the filing?

If you are following the generic [dissolution schema](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/dissolution.json) then the CP needs to be stored in the `parties` object (with `roleType` and `appointmentDate` set).

Reply:
Currently, I am not setting, data to send to API, I will do as per schema when i a m working on file/save draft

Reply:
ok